fe/fe-core/src/main/java/com/starrocks/statistic/StatisticsMetaManager.java

```diff
@@ -67,10 +70,27 @@ public class StatisticsMetaManager extends MasterDaemon {
                 new ColumnDef("min", new TypeDef(minType)),
                 new ColumnDef("update_time", new TypeDef(ScalarType.createType(PrimitiveType.DATETIME)))
         );
+
+        FULL_STATISTICS_COLUMNS = ImmutableList.of(
```

Comment:
why no db id?

Reply:
Because table_id is unique, db_id does not need to be used in the scenario of querying statistics and in the scenario of updating statistics